files/grest/rpc/transactions/tx_info.sql

```diff
@@ -371,7 +371,8 @@ BEGIN
             JSONB_BUILD_OBJECT(
               'policy_id', ENCODE(MA.policy, 'hex'),
               'asset_name', ENCODE(MA.name, 'hex'),
-              'quantity', MTM.quantity::text
```

Comment:
Should we retain `quantity` instead of `balance` for the amount minted?

Reply:
yeah, makes more sense for this endpoint

Reply:
In that case, does it make sense to rename all balance to quantity?

Reply:
Ok, yeah might be best to keep it consistent. I thing quantity works in all circumstances.

Reply:
+1 for renaming all `balance` to `quantity`